apps/omg_child_chain/test/omg_child_chain/integration/fixtures.exs

```diff
@@ -54,12 +54,19 @@ defmodule OMG.ChildChain.Integration.Fixtures do
         }
       })
 
-    default_file = Application.fetch_env!(:omg_child_chain, :fee_specs_file_name)
-    Application.put_env(:omg_child_chain, :fee_specs_file_name, file_name, persistent: true)
+    default_value = Application.fetch_env!(:omg_child_chain, :fee_adapter)
+
+    :ok =
+      Application.put_env(
```

Comment:
why do you need this set? are these not passed into the Fee server?

Reply:
We need it, it's passed by the supervisor because here the full ch-ch app is started

Reply:
wouldn't it be better if you set this in `config/test.exs`?

Reply:
File is prepared in this fixture under random name - needs to be here 
see also: https://github.com/omisego/elixir-omg/blob/2611eb1b29a3787882016a4d0d84ac140170bec2/apps/omg_child_chain/test/omg_child_chain/integration/fixtures.exs#L36

Reply:
so can you extract the file and put it into child chain so that it's fixed?

Reply:
It might be fixed by removing suport for flat files. Making a note to not forget https://github.com/omisego/elixir-omg/issues/1420